DigitalLibraryofTennessee/DLTN_XSLT
# Address DLTN_XSLT-185.

**GitHub Issue**: [DLTN_XSLT-185](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/187)

## What does this Pull Request do?

Adds manifests for new objects from ContentDM providers.

## What's new?

New manifest files and necessary changes to underlying transforms.

## How should this be tested?

Run this versus a record with a valid manifest and check if it gets a manifest link.

## Interested parties

@mlhale7 @CanOfBees
